copy/multiple.go

```diff
@@ -140,8 +162,34 @@ func (c *copier) copyMultipleImages(ctx context.Context, policyContext *signatur
 				UpdateDigest:    updatedManifestDigest,
 				UpdateSize:      int64(len(updatedManifest)),
 				UpdateMediaType: updatedManifestType})
-		default:
-			return nil, fmt.Errorf("copying image: invalid copy operation %d", instance.op)
+		case instanceCopyClone:
+			c.Printf("Replicating image %s (%d/%d)\n", instance.sourceDigest, i+1, len(instanceCopyList))
```

Comment:
When the `instanceCopyCopy` case starts with a `Debugf`, this should probably do the same thing.